superset/assets/javascripts/explorev2/stores/fields.js

```diff
@@ -993,15 +1004,15 @@ export const fields = {
   },
 
   viewport_latitude: {
-    type: 'IntegerField',
+    type: 'TextField',
```

Comment:
Create an issue with the information above:
https://github.com/airbnb/superset/issues/1912